### PR TITLE
Added possibility to invite new members via email in Team menu

### PR DESCRIPTION
**Pull request by Assstarte**
E2E tested.

### CODE REVIEW

**Comment by Assstarte** [comment]
@athlonUA Не забудь в БД добавить invite_hash колоночку в user_team.
Type: UUID
Nullable
Default: NULL

File: app/src/team/team.controller.ts [hunk 1]
```diff
@@ -48,6 +47,7 @@ export class TeamController {
 
     @Get(':id/rolecheck/:userId')
     async getTeamRole(@Response() res: any, @Param() param: any) {
+
```
Comment (athlonUA) [inline app/src/team/team.controller.ts]: remove nl

File: app/src/team/team.service.ts [hunk 2]
```diff
@@ -116,20 +117,22 @@ export class TeamService {
         }`;
 
         return new Promise((resolve, reject) => {
+            //@TODO: Generate random UUID
```
Comment (athlonUA) [inline app/src/team/team.service.ts]: remove

File: app/src/user/user.controller.ts [hunk 3]
```diff
@@ -55,6 +62,95 @@ export class UserController {
...
+        if (userExists.length > 0) {
+            //Such user is already registered
+            const invitedData: any = await this.teamService.inviteMemberToTeam(
+                body.userId,
+                body.teamId,
+                userExists[0].id
+            );
+            //Send an email:
+            const to = body.email;
+            //@TODO Add Team Name in the subject
```
Comment (athlonUA) [inline app/src/user/user.controller.ts]: Let's do this

File: app/src/user/user.controller.ts [hunk 4]
```diff
@@ -55,6 +62,95 @@ export class UserController {
...
+                createdData.data.insert_user.returning[0].id
+            );
+            //Send an email:
+            const to2 = body.email;
+            //@TODO Add Team Name in the subject
+            const subject2 = `You've been invited to the new team!`;
+            const html2 = `
+            Please follow the link below to accept the invitation to the team:     
+            <br /><br />
+            http://127.0.0.1:3000/team/${body.teamId}/invite/${
```
Comment (athlonUA) [inline app/src/user/user.controller.ts]: Replace with `${this.configService.get('API_URL')}`

File: app/.env.dist [hunk 5]
```diff
@@ -5,3 +5,5 @@ MAILER_PASSWORD = MAILER_PASSWORD
 MAILER_SMTP = MAILER_SMTP
 MAILER_SENDER_EMAIL = MAILER_SENDER_EMAIL
 MAILER_SENDER_NAME = MAILER_SENDER_NAME
+API_URL = https://api.wobbly.me/
+APP_URL = https://time.wobbly.me/
```
Comment (athlonUA) [inline app/.env.dist]: Add a new line at the EOF

**Pull request merged**